packages/math-input/src/components/keypad-context.ts

```diff
@@ -8,7 +8,7 @@
  */
 import * as React from "react";
 
-import type {RendererInterface} from "./types";
+import type {RendererInterface} from "@khanacademy/perseus";
```

Comment:
🤔  This has the unfortunate result though of creating a circular dependency (math-input depends on Perseus, Perseus depends on math-input). I'm not sure that's even allowed.

Reply:
> @jeremywiebe I'm wondering if shared types should be in perseus-core?

That was my gut feel too. Slowly move these types that need to be shared into perseus-core.  I don't want it to become a complete dumping ground, but I think this move makes sense. 👍

Reply:
So I moved what I needed to move into a types.ts folder in `perseus-core`. If you're still getting a bad vibe from this PR I don't mind just closing it out. Just makes sense to me that the KeypadContext should live closer to the Keypad.